docs/application/native/guides/ui/dali/actors-n.md

```diff
@@ -5,7 +5,9 @@
 
 An actor is the basic component that composes the entire scene. It can have visible (for example, UI components) or invisible (for example, a camera actor or layer) forms.
 
-An actor is also the primary object with which DALi applications interact. Multiple types of event signals provided by actors can be handled in an application through user-defined callback functions.
+An actor is also the primary object with which DALi applications interact. Controls and Layers all derive from Actor.
+
+Multiple types of event signals provided by actors can be handled in an application through user-defined callback functions.
 
```

Comment:
Multiple types of event signals provided by actors can be handled in an application through user-defined callback functions.

Please rephrase to:

In an application, multiple types of event signals provided by actors can be handled through user-defined callback functions.

Reply:
Okay, changed the description.